docs/control-structures.rst

```diff
@@ -284,10 +284,12 @@ Solidity internally allows tuple types, i.e. a list of objects of potentially di
         }
 
         function g() public {
-            // Declares and assigns the variables. Specifying the type explicitly is not possible.
-            var (x, b, y) = f();
-            // Assigns to a pre-existing variable.
-            (x, y) = (2, 7);
+            // Variables declared with type
+            uint x;
+            bool b;
+            uint y;
+            // These pre-existing variables can then be assigned to the tuple values
```

Comment:
I think this should either say `These pre-existing variables can then be assigned from the tuple values` or `Tuple values can be assigned to these pre-existing variables`.